docs/pages/button.js

```diff
@@ -259,245 +212,503 @@ card(
+    <MainSection.Subsection
+      columns={2}
+      title="States"
+      description={`
+1. Default
+    The typical state of a Button that represents it can be interacted with and is not in a selected state.
+2. Disabled
+Used to block user interaction such as hover, focus and click. Disabled Buttons are completely unreachable by a keyboard and screenreader, so do not attach Tooltips to disabled Buttons.
+3. Selected
+  When Button is used to toggle a boolean state or control the visibility of other elements (e.g. Dropdown), use the \`selected\` prop to indicate the current state. Do not use this prop with \`role="link"\` Buttons.
+`}
+    >
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Button
+  accessibilityLabel='Save'
+  color="red"
+  text="Save"
+  size="lg"
+/>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Button
+  accessibilityLabel='Submit'
+  disabled
+  text="Submit"
+  size="lg"
+/>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Button
+  accessibilityLabel='Following'
+  text="Following"
+  selected
+  size="lg"
+/>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection title="Accessibility props: controls, expanded, & popup">
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+function ActionDropdownExample() {
+  const [open, setOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState(null);
+  const anchorRef = React.useRef(null);
+  const onSelect = ({ item }) => setSelected(item);
   return (
-    <Button
-      selected={selected}
-      onClick={() => {setSelected(!selected)}}
-      text={selected ? "Selected" : "Deselected"}
-    />
+    <Flex justifyContent="center">
+      <Button
+        accessibilityControls="action-variant-dropdown-example"
+        accessibilityExpanded={open}
+        accessibilityHaspopup
+        iconEnd="arrow-down"
+        onClick={() => setOpen((prevVal) => !prevVal)}
+        ref={anchorRef}
+        selected={open}
+        size="lg"
+        text={selected ? selected.label : 'Display'}
+      />
+      {open && (
+        <Dropdown anchor={anchorRef.current} id="action-variant-dropdown-example" onDismiss={() => setOpen(false)}>
+          <Dropdown.Item
+            onSelect={onSelect}
+            option={{ value: 'Cozy', label: 'Cozy' }}
+            selected={selected}
+          />
+          <Dropdown.Item
+            onSelect={onSelect}
+            option={{ value: 'Comfy', label: 'Comfy' }}
+            selected={selected}
+          />
+        </Dropdown>
+      )}
+    </Flex>
   );
 }
 `}
-  />,
-);
-
-card(
-  <Example
-    id="ref"
-    name="ref"
-    defaultCode={`
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Ref"
+      description={`
+To control focus, or position any anchor components to IconButton, use ref, as shown in the example below.
```

Comment:
Run-on sentence, also missing formatting and referencing the wrong component.

"To control focus or position anchored components relative to Button, use `ref` as shown in the example below."

Reply:
Done!